### DISCUSSION TITLE
dbdev and testing

### DISCUSSION THREAD

**Discussion by MadMango** [supabase/supabase#18028] [Questions] [answered]
Hi everyone, not sure if I'm missing something, I tried using usebasejump/supabase-test-helpers.

The problem is that dbdev which is a dependency doesn't seem to be available on the local version of supabase.
But using it on the remote (linked) database is not that useful since I can't execute the test that I'm working on against a remote database.

Am I missing something here? It seems like I should either be able to have dbdev in the local supabase, or be able to execute tests against a remote db.

**Comment by silentworks** [comment, marked as answer]
At the moment dbdev isn't available in local setup, but you can install `userbasejump/supabase-test-helpers` without it. You can use the manual installation which is how it was being installed before dbdev was introduced, you can find the instructions here https://github.com/usebasejump/supabase-test-helpers#manual-installation

**Reply by MadMango** [reply]
That works, thank you!
